docs/data-sources/database.md

```diff
@@ -0,0 +1,107 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "encore_database Data Source - terraform-provider-encore"
+subcategory: ""
+description: |-
+  Encore provisioned database information
+---
+
+# encore_database (Data Source)
+
+Encore provisioned database information
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the Encore resource
+
+### Optional
+
+- `env` (String) The environment of the Encore resource. Defaults to the provider environment
+
+### Read-Only
+
+- `aws_rds` (Attributes) (see [below for nested schema](#nestedatt--aws_rds))
+- `gcp_cloud_sql` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_sql))
+
+<a id="nestedatt--aws_rds"></a>
+### Nested Schema for `aws_rds`
+
+Read-Only:
+
+- `arn` (String)
+- `parameter_group` (Attributes) (see [below for nested schema](#nestedatt--aws_rds--parameter_group))
+- `security_group` (Attributes) (see [below for nested schema](#nestedatt--aws_rds--security_group))
+- `subnet_group` (Attributes) (see [below for nested schema](#nestedatt--aws_rds--subnet_group))
```

Comment:
Same question as above regarding exposing `vpc`

Reply:
I suppose we could pull it higher. this is pretty much just how it's represented in the graph